src/api/common/schemas/ledger-options.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "ledger-options",
+  "description": "Options for getLedger",
+  "type": "object",
+  "properties": {
+    "ledgerVersion": {"$ref": "ledgerVersion"},
+    "includeAllData": {"type": "boolean"},
```

Comment:
While you're tinkering around, considering stuff in the area: includeAllData might be interpreted as a shorthand for includeLedgerState and includeTransactions